examples/devnet/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+version: '3.8'
+
+x-logging:
+  &default-logging
+  options:
+    max-size: '20m'
+    max-file: '3'
+  driver: json-file
+
+services:
+  lotus:
+    container_name: lotus
+    image: ${LOTUS_IMAGE}
+    # ports:
+    #   - "1234:1234"
+    environment:
+     - LOTUS_API_LISTENADDRESS=/dns/lotus/tcp/1234/http
+    restart: unless-stopped
+    logging: *default-logging
+    volumes:
+     - ./data/lotus:/var/lib/lotus:rw
+     - ./data/genesis:/var/lib/genesis:rw
+     - ./data/filecoin-proof-parameters/:/var/tmp/filecoin-proof-parameters:rw
```

Comment:
I think it'd be a good idea for this volume to point to `/var/tmp/filecoin-proof-parameters/` -- chances are that a developer has run a Filecoin devnet locally and already has the proof parameters necessary.

Furthermore I'd expect developers to remove often the `data` directory of their devnet (i.e. at least once a day), but we'd like to avoid having to download a few GiB of proof parameters daily.